fastapimsal/__init__.py

```diff
@@ -1,5 +1,5 @@
 try:
-    from importlib.metadata import version  # type: ignore
+    from importlib.metadata import version
```

Comment:
Ah, yes this now causes mypy to complain - it probably does need the ignore annotation. The full snippet is here:

https://github.com/OscartGiles/fastapimsal/blob/04a7b0b48d8b0f297809cddccc4fdbb13cf48bc7/fastapimsal/__init__.py#L1-L8

`version` was introduced in Python 3.8 so if you're using 3.7 mypy will fail to load it and raise an error, failing out tests. If you run with 3.8 or later mypy probably won't fail and everything will look fine (guessing you are running on 3.8 or later?).

Reply:
This wouldn't break our code though - it's only an issue for mypy.

Reply:
Yes, I was running on 3.9. Very happy to revert it, I had no reason to do it other than to get the tests to pass locally (mypy fails if there's a `type: ignore` that it considers unnecessary).

Reply:
Interesting. What is the error message it gives you?

Reply:
`fastapimsal/__init__.py:2: error: unused 'type: ignore' comment`

One way to silence it is to add `warn_unused_ignores = False` in `mypy.ini`. Not sure how bad that would be.

Reply:
Ah interesting. I think when you add `type: ignore` it gives the object the type `Any`.

But if you are on Python >3.8 mypy already has a type annotation for `version` (which it gets from [typeshed](https://github.com/python/typeshed)) and it then ignores our `type: ignore` (see https://mypy.readthedocs.io/en/stable/common_issues.html). Because we have the `--strict` flag in `mypy.ini` we say this should be an error.

I think your suggestion to add `warn_unused_ignores = False` makes sense. We're saying, if we can't find a type annotation make it type `Any`, otherwise use the type annotation.